finetune/ds_config_zero2.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I think we should unifiy json format to yaml, and rename this conf to `ds_config_zero2.yaml`

Reply:
Minming was following the same way how accelerate handles the deepspeed configuration. can you also take a look at accelerate and probably also other frameworks using Deepspeed and have a deeper analysis on the pros and cons?